src/QtImageReader.cpp

```diff
@@ -51,9 +60,70 @@ void QtImageReader::Open()
 	// Open reader if not already open
 	if (!is_open)
 	{
+
+#if USE_RESVG == 1
+		image = std::shared_ptr<QImage>(new QImage());
```

Comment:
You can also do `std::make_shared<QImage>();`. Thats the recommend way.

Reply:
I used this method for consistency since this is the way images are created elsewhere in the project source.

Reply:
I noticed that right after I made these comments. Tis up to you, see other comment for reasoning

Reply:
Okay.  I'm going to leave it as is for consistency.  Thanks for pointing this out, though!